packages/invoices/src/OutboundShipment/DetailView/columns.ts

```diff
@@ -156,36 +157,42 @@ export const useOutboundColumns = ({
           width: 180,
           getSortValue: row => {
             if ('lines' in row) {
-              return '';
+              const locations = row.lines
+                .map(({ location }) => location)
+                .filter(Boolean) as LocationRowFragment[];
+              return ifTheSameElseDefault(locations, 'name', '');
             } else {
-              return row.numberOfPacks ?? '';
+              return row.location?.name ?? '';
```

Comment:
I thought it was best to show the name of the location in the location column rather than the number of packs, but I can revert if you like

Reply:
I didn't even notice that it was showing numberOfPacks! why was it doing that??